core/src/main/java/tc/oc/pgm/action/ActionParser.java

```diff
@@ -414,6 +397,15 @@ public FillAction parseFill(Element el, Class<?> scope) throws InvalidXMLExcepti
         parser.parseBool(el, "events").orFalse());
   }
 
+  @MethodParser("team-alias")
+  public TeamAliasAction parseTeamAliasAction(Element el, Class<?> scope)
+      throws InvalidXMLException {
+    String teamId = Node.fromRequiredAttr(el, "team-id").getValue();
+    String alias = Node.fromRequiredAttr(el, "alias").getValue();
```

Comment:
Merely a style thing, but possibly fluent parsing could be used here? Not sure if using it is semantically equivalent and consistent with regular team ID + name parsing, however

Reply:
+1, could use fluent parsing

```suggestion
    String alias = parser.string(el, "alias").required();
```